parlai/tasks/cornell_movie/test/cornell_movie_valid.yml

```diff
@@ -3,168 +3,28 @@ acts:
-    - '  Hey, actually...'
-    - ' ...-onds...with just a fraction of what''s in...'
-    - ' No...'
-    - ' That''s why I didn''t give it...'
-    - ' Yeah...'
-    - '"A faithful heart makes wishes come true."'
-    - '"Age of Enlightenment". Shit. Like some waitress in a Las Vegas coffee shop
-      is going to get an obscure French philosophical reference. How demeaning. I
-      may as well have just said "Let me jump your ignorant bones."...'
-    - '"Are not capable of love" is what you mean.'
-    - '"At first I thought it was infatuation ...But oh it''s lasted so long..."'
-    - '"At the center of the investigation are well-known Washington-area attorneys
-      Robert Dean and Rachel Banks."'
-    - think that's, uh-
-    - uh uh.
-    - wHAT'S My status?! I've lost three men and your worthless fuck! After I kill
-      this asshole I'm coming your Yllo!
-    - was fascinated with you....  a real live detective....  You used to tell me
-      the most wonderful stories.  Were they true?
-    - we've got kidnapping, grand theft auto, burglary, and two counts of murder on
-      you, and I'm gonna see to it personally that it sticks!
-    - xxxxxx
-    - yeah, good, ok.
-    - you woke me up to tell me that?
-    - you're gonna like night clubs, they're really a lotta fun.
-    - '{baffled) I don''t follow you.'
-    reward: 0
     text: I figured you'd get to the good stuff eventually.
 - - episode_done: true
     eval_labels:
     - Like my fear of wearing pastels?
     id: cornell_movie
-    label_candidates:
-    - '  Hey, actually...'
-    - ' ...-onds...with just a fraction of what''s in...'
-    - ' No...'
-    - ' That''s why I didn''t give it...'
-    - ' Yeah...'
-    - '"A faithful heart makes wishes come true."'
-    - '"Age of Enlightenment". Shit. Like some waitress in a Las Vegas coffee shop
-      is going to get an obscure French philosophical reference. How demeaning. I
-      may as well have just said "Let me jump your ignorant bones."...'
-    - '"Are not capable of love" is what you mean.'
-    - '"At first I thought it was infatuation ...But oh it''s lasted so long..."'
-    - '"At the center of the investigation are well-known Washington-area attorneys
-      Robert Dean and Rachel Banks."'
-    - think that's, uh-
-    - uh uh.
-    - wHAT'S My status?! I've lost three men and your worthless fuck! After I kill
-      this asshole I'm coming your Yllo!
-    - was fascinated with you....  a real live detective....  You used to tell me
-      the most wonderful stories.  Were they true?
-    - we've got kidnapping, grand theft auto, burglary, and two counts of murder on
-      you, and I'm gonna see to it personally that it sticks!
-    - xxxxxx
-    - yeah, good, ok.
-    - you woke me up to tell me that?
-    - you're gonna like night clubs, they're really a lotta fun.
-    - '{baffled) I don''t follow you.'
-    reward: 0
     text: The "real you".
 num_episodes: 8310
-num_examples: 16759
+num_examples: 13914
```

Comment:
Oh wow

Reply:
Ya but the difference didn't have labels, so they weren't being used for metrics anyway.

Reply:
Ah interesting. So it was just eating compute but not directly affecting the model training. Presumably this was effectively reducing the batch size, then.